This is on top of the GCC 13 patches in our patchset.

https://bugs.gentoo.org/904342
--- a/src/3rdparty/chromium/third_party/blink/renderer/platform/graphics/dark_mode_types.h
+++ b/src/3rdparty/chromium/third_party/blink/renderer/platform/graphics/dark_mode_types.h
@@ -7,6 +7,8 @@
 
 namespace blink {
 
+#include <cstdint>
+
 enum class DarkModeResult : uint8_t {
   kDoNotApplyFilter,
   kApplyFilter,
